File: Java/topflytechCodec/src/com/topflytech/codec/entities/DeviceTempCollectionMessage.java
package com.topflytech.codec.entities;


import java.util.Date;
import java.util.List;

public class DeviceTempCollectionMessage extends Message{

    private int interval;
    private List<Float> tempList;
    private Date date;
    private int type = 1;//1:temp

    public int getType() {
        return type;
    }

    public void setType(int type) {
        this.type = type;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public int getInterval() {
        return interval;
    }

    public void setInterval(int interval) {
        this.interval = interval;
    }

    public List<Float> getTempList() {
        return tempList;
    }

    public void setTempList(List<Float> tempList) {
        this.tempList = tempList;
    }

}

File: TopflytechCodec/C#/TopflytechCodec/TopflytechCodec/Entities/DeviceTempCollectionMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TopflytechCodec.Entities
{
    public class DeviceTempCollectionMessage : Message
    {
        private int interval;
        private List<float> tempList;
        private DateTime date;
        private int type = 1; // 1:temp

        public int Type
        {
            get { return type; }
            set { type = value; }
        }

        public DateTime Date
        {
            get { return date; }
            set { date = value; }
        }

        public int Interval
        {
            get { return interval; }
            set { interval = value; }
        }

        public List<float> TempList
        {
            get { return tempList; }
            set { tempList = value; }
        }
    }
}
